src/nimony/semdata.nim

```diff
@@ -79,7 +79,7 @@ type
     types*: BuiltinTypes
     typeMem*: Table[string, TokenBuf]
     instantiatedTypes*: OrderedTable[string, SymId]
-    instantiatedProcs*: OrderedTable[string, SymId]
+    instantiatedProcs*: OrderedTable[SymId, OrderedTable[string, SymId]]
```

Comment:
Shouldn't instead the "comes from" symbol simply be part of the lookup key?

Reply:
As a tuple or in the string?